
#==============================================================================
# START!!! save_1111-1.py    PID: 2851    time: 2018-11-11 15:41:24.523815
#==============================================================================

  0%|          | 0/25 [00:00<?, ?it/s]100%|██████████| 25/25 [00:08<00:00,  2.91it/s]
rm: cannot remove '../data/tr_1111-1.pkl': No such file or directory
no dup :) 
X.shape (7848, 350)
  0%|          | 0/25 [00:00<?, ?it/s] 12%|█▏        | 3/25 [03:44<27:24, 74.77s/it] 20%|██        | 5/25 [07:29<29:56, 89.85s/it] 32%|███▏      | 8/25 [13:47<29:17, 103.38s/it] 36%|███▌      | 9/25 [17:35<31:15, 117.24s/it] 40%|████      | 10/25 [21:20<32:00, 128.05s/it] 44%|████▍     | 11/25 [23:52<30:22, 130.21s/it] 48%|████▊     | 12/25 [27:33<29:51, 137.82s/it] 60%|██████    | 15/25 [32:07<21:24, 128.48s/it] 64%|██████▍   | 16/25 [35:49<20:09, 134.35s/it] 68%|██████▊   | 17/25 [39:27<18:34, 139.25s/it] 72%|███████▏  | 18/25 [42:51<16:40, 142.87s/it] 76%|███████▌  | 19/25 [45:39<14:25, 144.17s/it] 96%|█████████▌| 24/25 [46:46<01:56, 116.96s/it]100%|██████████| 25/25 [46:56<00:00, 112.67s/it]
rm: cannot remove '../data/te_1111-1.pkl': No such file or directory

#==============================================================================
# SUCCESS !!! save_1111-1.py
#==============================================================================

time: 53.49min
Stopping instance(s) instance-1...
